spec/models/info_request_spec.rb

```diff
@@ -556,12 +556,15 @@
   describe "when using a plugin and calculating the status" do
 
     before do
-      InfoRequest.send(:require, File.expand_path(File.dirname(__FILE__) + '/customstates'))
-      InfoRequest.send(:include, InfoRequestCustomStates)
-      InfoRequest.class_eval('@@custom_states_loaded = true')
+      require File.expand_path(File.dirname(__FILE__) + '/customstates')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.